src/main/java/core/basesyntax/ForConstants.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class ForConstants {
+    public static final String LINE_SEPARATOR = "~!";
+    public static final String CSV_FILE_SEPARATOR = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final int VALUE_ZERO = 0;
+    public static final int VALUE_ONE = 1;
```

Comment:
Name these variables more informative or delete

Reply:
I renamed it.